packages/manager/apps/hub/src/dashboard/dashboard.constants.js

```diff
@@ -0,0 +1,286 @@
+  'club',
+  'coach',
+  'codes',
+  'coffee',
+  'com',
+  'community',
+  'company',
+  'computer',
+  'condos',
+  'construction',
+  'consulting',
+  'contact',
+  'contractors',
+  'cool',
+  'coupons',
+  'credit',
+  'creditcard',
+  'cruises',
+  'dance',
+  'dating',
+  'deals',
+  'degree',
+  'delivery',
+  'democrat',
+  'dental',
+  'dentist',
+  'design',
+  'diamonds',
+  'digital',
+  'direct',
+  'directory',
+  'discount',
+  'dk',
+  'doctor',
+  'dog',
+  'domains',
+  'education',
+  'email',
+  'energy',
+  'engineer',
+  'engineering',
+  'enterprises',
+  'equipment',
+  'estate',
+  'events',
+  'exchange',
+  'expert',
+  'exposed',
+  'express',
+  'fail',
+  'family',
+  'farm',
+  'finance',
+  'financial',
+  'fish',
+  'fitness',
+  'flights',
+  'florist',
+  'football',
+  'forex',
+  'forsale',
+  'foundation',
+  'fund',
+  'furniture',
+  'futbol',
+  'fyi',
+  'gallery',
+  'games',
+  'gifts',
+  'gives',
+  'glass',
+  'global',
+  'gmbh',
+  'gold',
+  'golf',
+  'graphics',
+  'gratis',
+  'green',
+  'gripe',
+  'group',
+  'guide',
+  'guru',
+  'haus',
+  'healthcare',
+  'hockey',
+  'holdings',
+  'holiday',
+  'hospital',
+  'house',
+  'immo',
+  'immobilien',
+  'industries',
+  'info',
+  'institute',
+  'insure',
+  'international',
+  'investments',
+  'irish',
+  'jetzt',
+  'jewelry',
+  'kaufen',
+  'kim',
+  'kitchen',
+  'land',
+  'lawyer',
+  'lease',
+  'legal',
+  'lgbt',
+  'life',
+  'lighting',
+  'limited',
+  'limo',
+  'live',
+  'loans',
+  'ltd',
+  'luxe',
+  'maison',
+  'management',
+  'market',
+  'marketing',
+  'markets',
+  'mba',
+  'media',
+  'memorial',
+  'mobi',
+  'moda',
+  'money',
+  'mortgage',
+  'movie',
+  'navy',
+  'network',
+  'news',
+  'ninja',
+  'one',
+  'organic',
+  'partners',
+  'parts',
+  'pet',
+  'photography',
+  'photos',
+  'pictures',
+  'pink',
+  'pizza',
+  'pl',
+  'place',
+  'plumbing',
+  'plus',
+  'poker',
+  'pro',
+  'productions',
+  'promo',
+  'properties',
+  'pub',
+  'recipes',
+  'red',
+  'rehab',
+  'reise',
+  'reisen',
+  'rentals',
+  'repair',
+  'report',
+  'republican',
+  'restaurant',
+  'reviews',
+  'rip',
+  'rocks',
+  'run',
+  'sale',
+  'salon',
+  'sarl',
+  'school',
+  'schule',
+  'services',
+  'sh',
+  'shiksha',
+  'shoes',
+  'shopping',
+  'show',
+  'singles',
+  'ski',
+  'soccer',
+  'social',
+  'software',
+  'solar',
+  'solutions',
+  'studio',
+  'style',
+  'supplies',
+  'supply',
+  'support',
+  'surgery',
+  'systems',
+  'tax',
+  'taxi',
+  'team',
+  'technology',
+  'tennis',
+  'theater',
+  'tienda',
+  'tips',
+  'tires',
+  'today',
+  'tools',
+  'tours',
+  'town',
+  'toys',
+  'trading',
+  'training',
+  'travel',
+  'university',
+  'vacations',
+  'ventures',
+  'vet',
+  'viajes',
+  'video',
+  'villas',
+  'vin',
+  'vision',
+  'vote',
+  'voto',
+  'voyage',
+  'watch',
+  'wine',
+  'works',
+  'world',
+  'wtf',
+  'xn--6frz82g',
+  'xyz',
+  'zone',
+];
```

Comment:
**question:** Can we avoid maintaining this list in the app? Better to get it from the API, since the list is huge.

Reply:
There is no api available to report repracing information.